### PR TITLE
print version arg

### PR DESCRIPTION
**Pull request by mattnibs**

### CODE REVIEW

**Comment by mattnibs** [comment]
Still need to add release.md file

**Comment by mattnibs** [comment]
Actually I'd like to keep this pr confined to just go release version. For js releases we'll have to update the package.json... I'd like to file this stuff in a separate pr I'm working on that collocates all our javascript code into a single sub directory.

File: Makefile [hunk 1]
```diff
@@ -18,10 +18,10 @@ test-system: build
 
 build:
 	@mkdir -p dist
-	@go build -o dist ./cmd/...
+	@go build -ldflags='-s -X main.version=$(shell ./scripts/version.sh)' -o dist ./cmd/...
```
Comment (nwt) [inline Makefile]: How about `git describe --always --broken --dirty --tags` for this? I think it's more informative.

Comment (mattnibs) [reply Makefile]: Doesn't work

Comment (mattnibs) [reply Makefile]: J/k, idk the output is really weird `v0.0.1-123-g21fa9bc-dirty`

**Pull request merged**